opentelemetry-api/src/opentelemetry/context/propagation/tracecontexthttptextformat.py

```diff
@@ -66,35 +68,30 @@ def extract(
         header = get_from_carrier(carrier, cls._TRACEPARENT_HEADER_NAME)
 
         if not header:
-            return trace.INVALID_SPAN_CONTEXT
+            return trace.generate_span_context()
```

Comment:
These changes get reverted if we go with your solution to #233 right?

Reply:
correct.